sql/core/src/test/scala/org/apache/spark/sql/sources/v2/TestInMemoryTableCatalog.scala

```diff
@@ -36,9 +36,13 @@ import org.apache.spark.sql.util.CaseInsensitiveStringMap
 
 // this is currently in the spark-sql module because the read and write API is not in catalyst
 // TODO(rdblue): when the v2 source API is in catalyst, merge with TestTableCatalog/InMemoryTable
```

Comment:
@rdblue, per this comment, I copied over namespace-related functions from `TestTableCatalog`. Please let me know this is not desirable.

Reply:
Sounds fine to me